lib/si-pkg/src/spec/attribute_value.rs

```diff
@@ -34,6 +35,14 @@ impl fmt::Display for KeyOrIndex {
         Ok(())
     }
 }
+impl Hash for KeyOrIndex {
```

Comment:
surprised you can't derive a hash for this type

Reply:
try adding #[derive(Hash)] and see if it works :)

Reply:
I don't know why I thought I couldn't ... it works fine!